app/src/store/bootstrap.ts

```diff
@@ -209,9 +210,25 @@ export const bootstrapCalls = createTypedAsyncThunk<
     }
 
     const calls = callsResult.value;
+    // Get all the call sids stored in async storage.
+    const storedCallSids = new Set(await AsyncStorage.getAllKeys());
     for (const call of calls.values()) {
       await dispatch(handleCall({ call }));
+
+      // If the call is still active, the native layer will still have it
+      // cached.
+      const callSid = call.getSid();
+      if (callSid) {
+        // Mark a call as still active, and therefore keep it in async storage.
```

Comment:
keep by using `delete` ?

Reply:
I suppose the comment is a bit strangely worded, but the idea is that if a call remains (i.e. it is _not_ deleted) then it is stale. We we mark it as active (not stale) by deleting it from the set of stale calls.